examples/specs/circle_flatten.vl.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "data":
+  {"values":
+   [{ "key": "alpha", "foo": [1, 2],    "bar": ["A", "B"] },
+    { "key": "beta",  "foo": [3, 4, 5], "bar": ["C", "D"] }
+   ]
+  },
+  "transform": [{"flatten": ["foo", "bar"]}],
+  "mark": "circle",
+  "encoding": {
+    "x": {
+
```

Comment:
1) Why line breaks here.  

2) I think you can fit field and type in one online for each of x, y, and color